projects/1.18/assets/dynamic-electricity/dynamicelectricity/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+  "block.dynamicelectricity.motorachv": "480V交流电动机",
+  "block.dynamicelectricity.motoraclv": "120V交流电动机",
+  "block.dynamicelectricity.motoracmv": "240V交流电动机",
+  "block.dynamicelectricity.motordchv": "480V直流电动机",
+  "block.dynamicelectricity.motordclv": "120V直流电动机",
+  "block.dynamicelectricity.motordcmv": "240V直流电动机",
+  "container.motorachv": "480V交流电动机",
+  "container.motoraclv": "120V交流电动机",
+  "container.motoracmv": "240V交流电动机",
+  "container.motordchv": "480V直流电动机",
+  "container.motordclv": "120V直流电动机",
+  "container.motordcmv": "240V直流电动机",
+  "fluid.dynamicelectricity.fluidlubricant": "工业润滑油",
+  "gui.dynamicelectricity.displayunit.industrialenergyunit.name": "工业能量单元",
+  "gui.dynamicelectricity.displayunit.industrialenergyunit.nameplural": "工业能量单元",
+  "gui.dynamicelectricity.displayunit.industrialenergyunit.symbol": "IE",
+  "gui.dynamicelectricity.motor.generating": "生成中",
+  "gui.dynamicelectricity.motor.lubricant": "润滑油：",
+  "gui.dynamicelectricity.motor.output": "输出：%s",
+  "gui.dynamicelectricity.motor.tier": "等级：%s",
+  "gui.dynamicelectricity.motor.usage": "使用：%s",
+  "gui.dynamicelectricity.motor.voltage": "电压：%s",
+  "gui.dynamicelectricity.motor.wattage": "功率：%s",
+  "guidebook.dynamicelectricity.chapter.acmotors": "交流电动机",
+  "guidebook.dynamicelectricity.chapter.acmotors.l1": "交流电动机能够将电动力学焦耳（J）转换为Forge能量单位（FE）。它能够以100%的效率实现这一点。存在三级电机，提供不同的吞吐率以满足您的要求。",
+  "guidebook.dynamicelectricity.chapter.dcmotors": "直流电动机",
+  "guidebook.dynamicelectricity.chapter.dcmotors.l1": "直流电动机能够将Forge能量单位（FE）转换为电动力学焦耳（J）。它以95%的效率实现了这一目标。与交流电动机不同，直流电动机需要一个名为导体刷的额外组件来运行。刷子的耐用性有限，但可以通过从电机顶部使用漏斗漏下更多来轻松补充。存在三层电机，提供不同的吞吐率以满足您的要求。",
```

```suggestion
  "guidebook.dynamicelectricity.chapter.dcmotors.l1": "直流电动机能够将Forge能量单位（FE）以95%的效率转换为电动力学焦耳（J）。与交流电动机不同，直流电动机需要一个名为导体刷的额外组件来运行。刷子的耐久度有限，但可以通过从电机顶部使用漏斗来轻松补充。存在三层电机，提供不同的吞吐率以满足您的需求。",
```
疑似机翻，好多不通顺的地方
3 tiers翻成三层？

上下文通查吧